tests/integration/helpers/cluster.py

```diff
@@ -1132,14 +1624,20 @@ def http_query_and_get_error(self, sql, data=None, params=None, user=None, passw
     def stop_clickhouse(self, stop_wait_sec=30, kill=False):
         if not self.stay_alive:
             raise Exception("clickhouse can be stopped only with stay_alive=True instance")
+        try:
+            ps_clickhouse = self.exec_in_container(["bash", "-c", "ps -C clickhouse"], user='root')
```

Comment:
`pkill -0` is simpler

Reply:
Probably yes. I have seen undead clickhouse processes. And I'm not sure that there was still a pid file.